shuffle/genesis/src/lib.rs

```diff
@@ -69,11 +73,12 @@ pub fn generate_validator_config(
     Ok(validators.pop().unwrap())
 }
 
+/// TODO: remove this code and use Move package manager to build sources when it is available
 pub fn build_move_sources() -> Result<()> {
     // Build the Move code to ensure we get the latest changes in script builders + the genesis WriteSet
     utils::time_it("Building Move code", || {
-        let output = Command::new("df-cli")
-            .args(&["sandbox", "publish"])
+        let output = Command::new("move")
+            .args(&["sandbox", "publish", "--mode=bare"])
```

Comment:
Do we have any code comments or usage documentation that shares the `bare` mode being one of the minimal sets?

Reply:
Documentation for move-cli describes modes: https://github.com/diem/diem/tree/main/language/tools/move-cli#using-the-cli-with-modes-and-genesis-state

We are using `bare` mode here because we don't need to compile code other than the user-defined modules (`SampleModule` in this case). The binaries/compiled artifacts for the other modules (stdlib + DF) are already included in Diem Framework artifacts. This approach can be improved though, with @tzakian's awesome Move package manager when it becomes fully available.